axelrod/strategies/axelrod_second.py

```diff
@@ -1300,11 +1300,11 @@ def strategy(self, opponent: Player) -> Action:
         if self.burned or random.random() > self.prob:
             # Tit-for-Tat with probability 1-`prob`
             return self.try_return(opponent.history[-1], inc_parity=True)
-        else:
-            # Otherwise Defect, Cooperate, Cooperate, and increase `prob`
-            self.prob += 0.05
-            self.more_coop, self.last_generous_n_turns_ago = 2, 1
-            return self.try_return(D, lower_flags=False)
+        
+        # Otherwise Defect, Cooperate, Cooperate, and increase `prob`
+        self.prob += 0.05
+        self.more_coop, self.last_generous_n_turns_ago = 2, 1
+        return self.try_return(D, lower_flags=False)
 
```

Comment:
What happened here? This doesn't look like it should have changed.

Reply:
Sorry, maybe I shouldn't have slipped this in here.  I got rid of a useless else.